src/generator/AutoRest.Go/Templates/VersionTemplate.cshtml

```diff
@@ -36,12 +36,12 @@ const (
 
 @EmptyLine
 // UserAgent returns the UserAgent string to use when sending http.Requests.
-func @(Model.UpdateNameIfDuplicate("UserAgent", Model.PackageName))() string {
-    return fmt.Sprintf(userAgentFormat, @(Model.UpdateNameIfDuplicate("Version", Model.PackageName))(), "@(Model.PackageName)", "@(Model.ApiVersion)")
+func UserAgent() string {
```

Comment:
okay. 

---
In reply to: [77408984](https://github.com/Azure/autorest/pull/1397#discussion_r77408984) [](ancestors = 77408984)